sn_node/src/put_validation.rs

```diff
@@ -425,11 +453,14 @@ impl Node {
 
         // unpack transfer
         trace!("Unpacking incoming Transfers for record {pretty_key}");
-        let (cash_notes, transfer_for_us) = self
+        let (cash_notes, royalties_cash_notes) = self
             .cash_notes_from_payment(&payment, &wallet, pretty_key.clone())
             .await?;
 
-        info!("{:?} cash notes are for us", cash_notes.len());
+        info!(
+            "{} cash notes are for us for {pretty_key}",
+            cash_notes.len()
```

Comment:
shall we also log the sum value ?

Reply:
agreed

Reply:
actually we already log the cash notes value later on when calculating received_fee